src/core/Toast/Toast.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component, forwardRef, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { baseStyles } from './Toast.baseStyles';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { Icon } from '../Icon/Icon';
+import { Heading } from '../Heading/Heading';
+import { HtmlDiv, HtmlDivWithRef } from '../../reset';
+import { hLevels } from '../../reset/HtmlH/HtmlH';
+
+export interface ToastProps {
+  /** Set aria-live mode for the Toast text content and label.
+   * @default 'polite'
+   */
+  ariaLiveMode?: 'polite' | 'assertive' | 'off';
+  /** Label heading for the Toast */
+  headingText?: string;
+  /** Main content of the Toast */
+  children?: ReactNode;
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** Elements to be rendered under the Toast text. Can be e.g. buttons, links etc. */
+  headingVariant?: Exclude<hLevels, 'h1'>;
+  /** Id for focus */
+  id?: string;
+}
+interface InnerRef {
+  forwardedRef: React.RefObject<HTMLDivElement>;
+}
+const baseClassName = 'fi-toast';
+export const toastClassNames = {
+  styleWrapper: `${baseClassName}-wrapper`,
+  heading: `${baseClassName}-heading`,
+  contentWrapper: `${baseClassName}-content-wrapper`,
+  icon: `${baseClassName}_icon`,
+  iconWrapper: `${baseClassName}_icon-wrapper`,
+};
+class BaseToast extends Component<ToastProps & InnerRef> {
+  render() {
+    const {
+      ariaLiveMode = 'polite',
+      children,
+      className,
+      forwardedRef,
```

Comment:
Why is `forwardedRef` deconstructed and never used?
Shouldn't it be allowed to be inside the `passProps` so that it will continue it's journey to **HtmlDivWithRef**?

Reply:
Should be fixed.

Reply:
Seems to be still same.

Reply:
fixed